assets/reviews.js

```diff
@@ -1,40 +1,44 @@
 (async () => {
-  const reviewsContainer = document.querySelector('.yc-product-reviews');
-  const reviewsWrapper = document.querySelector('.yc-reviews-wrapper');
-  const noDataSetter = (element) => {
-    if (reviewsContainer) {
-      reviewsContainer.remove();
-    }
-  };
+  const reviewsContainer = $('.yc-product-reviews');
+  const reviewsWrapper = $('.yc-reviews-wrapper');
+  const showMoreButton = $('#show-more');
+  let reviews = [];
+
+    const noDataSetter = (element) => {
+      if (reviewsContainer) {
+        reviewsContainer.remove();
+      }
+    };
+
+  const createReviewItem = (review) => {
+    const reviewItem = document.createElement('li');
+
+    reviewItem.classList.add('review-item');
+    createElement(reviewItem, review);
+    return reviewsWrapper.appendChild(reviewItem);
+  }
 
   try {
-    const reviews = await youcanjs.product.fetchReviews(productId).data();
+    const res = youcanjs.product.fetchReviews(productId, { limit: 3 });
```

Comment:
you forgot `async` here

Reply:
we have already a global async scope

Reply:
I mean isn't `youcajs.product.fetchReviews` an asynchrounous task?
why you remove `await` here?
const res = await youcanjs.product.fetchReviews(productId, { limit: 3 });